_posts/languages/nodejs/2000-01-01-common-errors.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Node.js Common Errors
+nav: Common Errors
+modified_at: 2020-06-12 00:00:00
+tags: nodejs
+index: 3
+---
+
+You may experience an error in your Node.js application that many customers
+faced when first deploying such application on Scalingo. Here is a list of the
+most common error messages.
+
+## devDependencies Also Contain the Build Dependencies
+
+The `devDependencies` section of the package.json file contains both development
+dependencies and build dependencies. By default Scalingo deployments install the
+dependencies from the `dependencies` section of the package.json file. It may
+lead to error messages such as `ng: not found` or `nest: not found`. In such
+situation, you have a couple of solutions:
+
+- Install all `devDependencies` ([doc]({% post_url
```

Comment:
I would say it would be better to include all interesting informations right here instead of linking them. WDYT?

Reply:
Do you think what I added is enough? I don't want to give too much information in this list.

Reply:
As the goal is to help users-that-do-not-read-doc deploy their apps on Scalingo, I think it would be better to not spread them in various page.

This doc page should be the one size fits all documentation page used by the support every time a NodeJS does not boot.